### ISSUE TITLE
Bsp mode always starts horizontally 

### ISSUE THREAD

**Issue by joelbrewster** [koekeishiya/kwm#280]
Hello,

If I use bsp in vertical mode, change to monocle then change back to bsp mode, bsp mode is horizontal and not vertical.
I had a quick look at the new rc file and didn't see anything about default orientation with bsp mode.
It appears bsp mode is always started horizontally.

**Comment by chrishoage** [comment]
kwm defaults to the golden ratio to determine which direction to split. You can override the ratio with the following command.

```
kwmc config optimal-ratio
```

You can also toggle the split mode for the focused container

`kwmc window -c split-mode toggle` 

this toggles between horizontal and vertical 

https://github.com/koekeishiya/kwm/tree/master/kwmc

**Comment by koekeishiya** [comment]
You can tell Kwm to always do a vertical / horizontal split with the following command:
`kwmc display -c vertical`or `kwmc display -c horizontal`.
The default value can be restored with: `kwmc display -c optimal`.

The above command currently applies to all displays, but will be made display specific in the future.

**Comment by joelbrewster** [comment]
I don't want kwm to always stay vertical, horizontal or change the ratio.
I think it should change back to whatever the user was using before the mode shift.
Example:

- start in bsp mode - **vertical** (split-mode toggle to vertical)
- change to monocle (space -t monocle)
- change back to bsp - **horizontal** (kwm changes back to horizontal and not what the user had selected before: **vertical**)

Basically, it's not remembering what the user had previously in any mode change. 
Resize a border, change modes, then change back; reverted to default of that mode, not what the user had arranged/moved/resized before.

Is this the intended behavior?

**Comment by koekeishiya** [comment]
That is the intended behaviour, yeah.
